app/src/pages/inside/settingsPage/settingTabs/notificationsTab/notificationsTab.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  updateProjectEmailConfig,
+  projectEmailCasesSelector,
+  projectEmailEnabledSelector,
+} from 'controllers/project';
+import { OWNER } from 'common/constants/permissions';
+import { activeProjectRoleSelector, userAccountRoleSelector } from 'controllers/user';
+import classNames from 'classnames/bind';
+import { canConfigreEmailNotifications } from 'common/utils/permissions';
+import { EmailEnableForm, EmailCaseForm } from './forms';
+import styles from './notificationsTab.scss';
+
+const cx = classNames.bind(styles);
+
+@connect(
+  (state) => ({
+    projectRole: activeProjectRoleSelector(state),
+    userRole: userAccountRoleSelector(state),
+    projectEmailConfig: {
```

Comment:
I think it's better to move `emailEnabled`, `emailCases` one level upper (without `projectEmailConfig`), since creating new object forces connected component to make unnecessary rerender